src/utils/removeTypename.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const omitTypename = (key: string, value: any) => {
```

Comment:
Using any here? unknown would be better of you don't have a type, then add further checks https://rahuulmiishra.medium.com/typescripts-any-vs-unknown-vs-never-when-and-how-to-use-them-35d81ea57c01

Reply:
Can also create generic types: https://www.typescriptlang.org/docs/handbook/2/generics.html